service.rb

```diff
@@ -1,24 +1,22 @@
-require 'socket'
 require 'cgi'
 require 'uri'
 require 'yaml/store'
+require 'rack/handler/puma'
+require 'rack'
```

Comment:
I encounter this error without that line: 
```
Traceback (most recent call last):
	8: from mirth.rb:4:in `<main>'
	7: from mirth.rb:4:in `new'
	6: from /Users/mapleong/src/github.com/wildmaples/mirth/service.rb:58:in `initialize'
	5: from /Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/rack/handler/puma.rb:63:in `run'
	4: from /Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/rack/handler/puma.rb:32:in `config'
	3: from /Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/rack/handler/puma.rb:32:in `new'
	2: from /Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/puma/configuration.rb:154:in `initialize'
	1: from /Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/puma/configuration.rb:161:in `configure'
/Users/mapleong/.gem/ruby/2.6.0/gems/puma-5.2.0/lib/rack/handler/puma.rb:34:in `block in config': uninitialized constant Rack::CommonLogger (NameError)
```

Reply:
Ah, good point, this is because of the `Verbose: true` that I suggested you add. Arguably this is a bug in Puma but, sure, this is now a good reason to `require 'rack'`!